### DISCUSSION TITLE
Hypernetwork tabs that could be added or fixed

### DISCUSSION THREAD

**Discussion by clacheek** [vladmandic/sdnext#467] [Ideas]
~~Nowadays, the hypernetwork tabs (when you have "sd_hypernetwork" on the user interface>quicksettings list) does not have an option to turn-off activated hypernetworks or a "none/empty" option, and it should be added or fixed since there's no way to turn HN's off when you pick one (a1111 also does not have this function, it had in the past but it got removed idk why XDD)...~~**(Added/Fixed)** Also it would be cool to have a way to control the weights by adding an slider (kinda difficulty one but well...).

"Then why? just use the hypernetwork in the prompt and change weights just like LoRA's!" 
Yeah, but personally I found it easier and faster to experiment by using the tabs ~~(in the past cause today it's a pain, i don't even use HN's anymore because of this).~~

Thanks and sorry if this is on the wrong discussion topic.

images to give some context
![firefox_tCRGQY2AkS](https://user-images.githubusercontent.com/114445644/234124091-f4b28b7d-aef1-4619-a0f0-1c345ba4eb58.png)
![firefox_fFiQwWgZgb](https://user-images.githubusercontent.com/114445644/234124094-1042a3bc-1446-434d-b35a-a1417b771fde.png)
![firefox_5brMvElynZ](https://user-images.githubusercontent.com/114445644/234124463-fa11700f-f4c4-421e-a7f2-209c26365d16.png)

**Comment by vladmandic** [comment]
> Thanks and sorry if this is on the wrong discussion topic.

Nope, its not - its a valid suggestion - I'll add it to backlog, but if someone wants to contribute, even better.
